tex/luametatex/files/patch-fix-gcc-build.diff

```diff
@@ -0,0 +1,9 @@
+--- cmake/luametatex.cmake	2024-03-11 16:34:51.000000000 +0800
++++ cmake/luametatex.cmake	2024-06-22 04:18:37.000000000 +0800
+@@ -86,5 +86,5 @@
+     # -g -S -d : remove all debugging symbols & sections
+     # -x       : remove all non-global symbols
+     # -X       : remove any compiler-generated symbols
+-    add_custom_command(TARGET luametatex POST_BUILD COMMAND ${CMAKE_STRIP} -g -S -d -x luametatex${CMAKE_EXECUTABLE_SUFFIX})
++    # add_custom_command(TARGET luametatex POST_BUILD COMMAND ${CMAKE_STRIP} luametatex${CMAKE_EXECUTABLE_SUFFIX})
```

Comment:
If you're commenting this out, why change the text of the line?

Reply:
@pmetzger Fair enough, that is unneeded. (I tried dropping args sequentially and it still kept failing.)

Reply:
@pmetzger Done